src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -85,15 +88,15 @@ public static IAEItemStack loadItemStackFromNBT( final NBTTagCompound i )
 			return null;
 		}
 
-		final AEItemStack item = AEItemStack.create( itemstack );
+		final AEItemStack item = AEItemStack.fromItemStack( itemstack );
 		item.setStackSize( i.getLong( "Cnt" ) );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "item" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2259)